addon/lib/toolbar-button.js

```diff
@@ -8,17 +8,20 @@ const {Panel} = require('sdk/panel');
 const querystring = require('sdk/querystring');
 const store = require('sdk/simple-storage').storage;
 const tabs = require('sdk/tabs');
+const self = require('sdk/self');
 
 const Mustache = require('mustache');
 const templates = require('./templates');
 Mustache.parse(templates.installed);
 Mustache.parse(templates.experimentList);
 
+const xulcss = require('./xulcss');
+xulcss.addXULStylesheet(self.data.url('button.css'));
```

Comment:
Doesn't it do this automatically here? https://github.com/dannycoates/testpilot/blob/85d9fd2be75aa4264497843265a2ca74fc73e9d1/addon/lib/xulcss.js#L42